test/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "@4c/tsconfig/node",
+  "compilerOptions": {
+    "noImplicitAny": false,
```

Comment:
I think achieveing `noImplicitAny` is actually easier than it seems and has big benefits! ok doing it in a separate PR though

Reply:
This is specifically for tests. The issue I was having is that createNodeRype doesn’t really export the class type definition for NodeType. I added an interface to it but figured we can be less strict for tests

Reply:
I think best practice for tests is to have a separate tsconfig.json and separate entrypoint altogether (otherwise you can do eg `describe` in your main app)

Reply:
wait. this is on test. ok never mind :P